lib/src/analyzer_impl.dart

```diff
@@ -122,8 +125,13 @@ class AnalyzerImpl {
   void prepareErrors() {
     for (Source source in sources) {
       context.computeErrors(source);
+
       var sourceErrors = context.getErrors(source);
       errorInfos.add(sourceErrors);
+
+      if (options.strongMode) {
+        errorInfos.add(strongChecker.computeErrors(source));
```

Comment:
One thing I noticed here, is that this is basically like ErrorVerifier: a set of errors computed after resolution. DDC is not currently running ErrorVerifier: https://github.com/dart-lang/dev_compiler/issues/25, but it should. This made me wonder if our "strong checker" pass should simply build on ErrorVerifier, and report its errors in the same bucket. That would remove this code, and remove the StrongChecker type entirely. Instead, it would simply be another factory that plugs into the AnalysisContext at startup, analogous to how DDC's inference plug-in works.

Thoughts?